### DISCUSSION TITLE
Switch group? like checkbox group

### DISCUSSION THREAD

**Discussion by Toshinaki** [mantinedev/mantine#2058] [Feature requests]
Since switch is a special form of checkbox, it'd be great to have a `Switch.Group` just like the `Checkbox.Group`.

Or is it possible to do:
```
<Checkbox.Group>
  <Switch />
  ...
</Checkbox.Group>
```

**Comment by rtivital** [comment]
I've added it to the backlog, it will be added in one of 5.x releases
